pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -66,16 +66,18 @@ const (
 	DefaultMonCount = 3
 	// MaxMonCount Maximum allowed mon count for a cluster
 	MaxMonCount = 9
-	// Msgr2port is the listening port of the messenger v2 protocol
-	Msgr2port int32 = 3300
+
+	// DefaultMsgr1Port is the default port Ceph mons use to communicate amongst themselves.
```

Comment:
```suggestion
	// DefaultMsgr1Port is the default port Ceph mons use to communicate amongst themselves prior to Nautilus
```

The current comment is a bit misleading, as of Nautilus monitors listen on both 6789 and 3300 so they use these 2 to communicate amongst themselves.